docs/docs/guides/labs/dg/multiple-projects.md

```diff
@@ -17,42 +17,40 @@ If you're just getting started, we recommend [scaffolding a single project](/gui
 
 If you need to collaborate with multiple teams, or work with conflicting dependencies that require isolation from each other, you can scaffold a workspace directory that contains multiple projects, each with their own separate Python environment.
 
-A workspace directory contains a root `dg.toml` with workspace-level settings, and a `projects` directory with one or more projects.
+A workspace directory contains a root `dg.toml` with workspace-level settings, and a `projects` directory with one or more projects. It also contains a Python environment in a `deployments/local` folder that can be used for running `dg` commands locally against the workspace.
 
-:::note
-
-A workspace does not define a Python environment by default. Instead, Python environments are defined per project.
+When a `dg` command runs in a workspace, it will create a subprocess for each project using that project's virtual environment. The diagram below demonstrates a workspace with two projects, as well as their virtual environments.
 
-:::
+![Diagram showing the virtual environments used by a workspace and 2 projects](/images/guides/build/projects-and-components/setting-up-a-workspace/workspace-venvs.png)
 
 ## Scaffold a new workspace and first project
 
-To scaffold a new workspace called `dagster-workspace`, run `uvx create-dagster workspace`:
+To scaffold a new workspace called `dagster-workspace`, run `uvx create-dagster workspace` and respond yes to the prompt to run `uv sync` after scaffolding:
 
 <CliInvocationExample path="docs_snippets/docs_snippets/guides/dg/workspace/1-dg-scaffold-workspace.txt" />
 
-Now we'll create a project inside our workspace called `project-1`. Run `uvx create-dagster project`. You will be prompted for the name of the project:
+The scaffolded workspace includes a `projects` folder, which is currently empty, and a `deployments` folder, which includes a `local` folder with a `pyproject.toml` file that specifies an environment for running `dg` commands locally against your workspace.
 
-<CliInvocationExample path="docs_snippets/docs_snippets/guides/dg/workspace/2-dg-scaffold-project.txt" />
+Next, enter the directory and activate the virtual environment for the `local` environment:
```

Comment:
So after the `source deployments/local/.venv/bin/activate` how does that virtual environment get associated with the workspace? Meaning `dg` when executed at workspace root will utilize that environment. Does the user always have to remember running `source deployments/local/.venv/bin/activate`? I think it is important to explain how this works.